### PR TITLE
Remove build config - soft approach

### PR DESCRIPTION
**Pull request by gianluz**

### CODE REVIEW

File: Dangerfile.main.kts [hunk 1]
```diff
@@ -24,4 +24,6 @@ if (danger.git.createdFiles.size + danger.git.modifiedFiles.size - danger.git.de
 
 if (danger.github!!.pullRequest.title.contains("WIP" ,false)) {
     warn("PR is classed as Work in Progress")
-}
\ No newline at end of file
+}
+
+warn("Test")
```
Comment (f-meloni) [inline Dangerfile.main.kts]: You forgot to remove this one :)

File: danger-kotlin-library/dependencies.gradle [hunk 2]
```diff
@@ -3,6 +3,6 @@ apply from: file('../dependencyVersions.gradle')
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     utilsDependencies()
-    kotlinDependencies()
+    //kotlinDependencies()
```
Comment (f-meloni) [inline danger-kotlin-library/dependencies.gradle]: Commented code :P

**Comment by f-meloni** [comment]
The user we are using has no verified email :( https://travis-ci.org/danger/kotlin/jobs/600092968#L825

File: danger-kotlin-library/dependencies.gradle (hunk 2 above)
Comment (gianluz) [reply danger-kotlin-library/dependencies.gradle]: yeah i'm not finish with my tests

File: Dangerfile.main.kts (hunk 1 above)
Comment (gianluz) [reply Dangerfile.main.kts]: added with purpose.. where is it the danger comment??

Comment (gianluz) [reply Dangerfile.main.kts]: added with purpose. where is it the danger comment?

File: danger-kotlin-library/dependencies.gradle (hunk 2 above)
Comment (gianluz) [reply danger-kotlin-library/dependencies.gradle]: yeah i know :) not finished with my tests

File: Dangerfile.main.kts (hunk 1 above)
Comment (f-meloni) [reply Dangerfile.main.kts]: https://github.com/danger/kotlin/pull/39#issuecomment-544179125

**Pull request merged**